tests/test_unit/test_cli/test_build_deploy_snapshots/cdf_infield_second_location.yaml

```diff
@@ -443,13 +437,10 @@ Transformation:
       version: '1'
     instanceSpace: sp_asset_oid2_source
     type: instances
-  destinationOidcCredentials:
-    audience: https://bluefield.cognitedata.com
+  destinationNonce:
```

Comment:
not sure here, there's code to issue warning on using this parameter, is it still in use?